cmake/FindWinHID.cmake

```diff
@@ -219,6 +221,7 @@ else()
 		${MINGWSEARCH_INCLUDE_DIRS}
 		/mingw
 		PATH_SUFFIXES
+		x86_64-w64-mingw32/include
```

Comment:
why is that needed ? here with mingw-w64 it works fine as-is

Reply:
I suspect it is a leftover from an older version of MinGW? I would also expect that the compiler should add its own include directory automatically.

Reply:
This could be removed, IMO - feel free to remove the line in your PR if it works without it.